paramak/shape.py

```diff
@@ -855,14 +856,27 @@ def export_svg(self, filename: str) -> str:
 
         return str(path_filename)
 
-    def export_html(self, filename: str):
+    def export_html(
+            self,
+            filename: str = "shape.html",
+            facet_splines: bool = True,
+            facet_circles: bool = True,
+            tolerance: float = 1e-3,
+    ):
         """Creates a html graph representation of the points and connections
         for the Shape object. Shapes are colored by their .color property.
         Shapes are also labelled by their .name. If filename provided doesn't
         end with .html then .html will be added.
 
         Args:
-            filename (str): the filename used to save the html graph
+            filename: the filename used to save the html graph. Defaults to
+                shape.html
+            facet_splines: If True then spline edges will be faceted. Defaults
+                to True.
+            facet_splines: If True then circle edges will be faceted.Defaults
+                to True.
+            tolerance: faceting toleranceto use when faceting cirles and
+                splines. Defaults to 1e-3.
```

Comment:
- the types are missing
- `facet_splines `is written twice, second one needs to be replaced by `facet_circles` I suppose ?